masonry_core/src/core/contexts.rs

```diff
@@ -1178,6 +1197,80 @@ impl_context_method!(
             self.global_state.mutate_callbacks.push(callback);
         }
 
+        /// Get direct reference to the stored child, and a context handle for that child.
+        ///
+        /// This context lets you set pass flags for the child widget, which may be tricky.
+        /// In general, you should avoid setting the flags of a pass that runs before the
+        /// pass you're currently in.
+        ///
+        /// See [pass documentation](crate::doc::doc_05_pass_system) for the pass order.
+        pub fn get_raw<Child: Widget + FromDynWidget + ?Sized>(
+            &mut self,
+            child: &mut WidgetPod<Child>,
+        ) -> (&Child, RawCtx<'_>) {
```

Comment:
I would expect the raw context here to only be able to read flags.  This is the motivation for my suggestion about `WidgetMut` below.